gnosis/cowswap
# Adds useParseReferralQueryParam hook to parse the referral link from the url.

# Summary

Closes #852

Adds a `useParseReferralQueryParam` hook to parse the referral link from the url.

# Testing steps

1. Open the app
[ ] In the top menu, there should be a link to the `Rewards` page
2. Click on `Rewards`
[ ] It should lead you to the rewards page
3. Click on the `Connect wallet` button if no wallet is connected. Otherwise click on the `Create affiliate link` button
[ ] The referral link should've been printed on the browser's console
4. Copy and paste the referral link in a new browser's window/tab address bar
[ ] The referral link should've been printed on the browser's console
